views/partials/base.tmpl

```diff
@@ -3,13 +3,16 @@
 <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
-    <title>{{block "title" .}}{{.PageContext.LongName}}{{end}}</title>
+    <title>{{block "title" .}}{{.PageContext.LongName}}{{end}}</title>    
 
-    <link rel="stylesheet" href="/css/bootstrap-3.3.7.min.css" type="text/css" />
-    <link rel="stylesheet" href="/css/main.scss.css" type="text/css" />
+    <script src="/js/jquery-3.1.1.min.js"></script>
```

Comment:
Why have these been moved to the top? Better to load them after the rest of the page